### PR TITLE
fix(ui): error highlighting on selections

### PR DESCRIPTION
**Pull request by insmac**
Fixes https://github.com/questdb/questdb/issues/1964

This fix ensures proper error marking for queries executed both at cursor position and selection:
CleanShot 2022-03-18 at 14 12 14@2x

By fine tuning the default word pattern regex provided by Monaco it also fixes errors in table names enclosed by single quotes:
CleanShot 2022-03-18 at 14 12 01@2x

### CODE REVIEW

File: ui/src/scenes/Editor/Monaco/questdb-sql/conf.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import * as monaco from "monaco-editor"
 
 export const conf: monaco.languages.LanguageConfiguration = {
+  wordPattern: /(-?\d*\.\d\w*)|([^\`\~\!\@\#\$\%\^\&\*\(\)\-\=\+\[\{\]\}\\\|\;\:\"\,\<\>\/\?\s]+)/g,
```
Comment (argshook) [inline ui/src/scenes/Editor/Monaco/questdb-sql/conf.ts:4]: would be helpful to add a comment briefly explaining the purpose of this regex

Comment (insmac) [reply ui/src/scenes/Editor/Monaco/questdb-sql/conf.ts:4]: @argshook 
This is a derivative of default one (if not provided explicitly), like so:
https://github.com/microsoft/vscode/blob/main/src/vs/editor/common/core/wordHelper.ts#L31

The difference being we're allowing `'` and `.' to be included for the string to be considered a "word". 
In short, a default implementation won't consider `'table_name'` a word and thus won't highlight it if needed, whereas this change will make it work. The dot is also needed for table names created from CSV imports, for example.

Comment (argshook) [reply ui/src/scenes/Editor/Monaco/questdb-sql/conf.ts:4]: oh, i wasn't clear enough, i meant a comment in the code, for future generations :)

Comment (insmac) [reply ui/src/scenes/Editor/Monaco/questdb-sql/conf.ts:8]: @argshook Done 📝

**Review by argshook — APPROVED** [review]

**Pull request merged**